api/api.yaml

```diff
@@ -191,6 +191,26 @@ paths:
           $ref: '../result/request.yaml#/components/responses/BadRequest'
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
+  
+  /control/actions:
+    post:
+      tags: ['Control']
+      operationId: set_action
+      description: >-
+        Set specific action/trigger on configuration resources on the traffic generator.
```

Comment:
This should be limited to specific state/actions on device configuration resources

Reply:
done